app/client/views/confirmation/confirmation.html

```diff
@@ -115,7 +115,7 @@
             <p>
               We will have a hardware checkout, and we'll try our best to provide things that would be useful to you - however, we can't guarantee everything. We will publish a list of available hardware closer to the day of the event.
             </p>
-            <input type="text" ng-model="user.confirmation.hardware">
+            <input type="text" ng-model="user.confirmation.hardwareRequest">
```

Comment:
Maybe don't change this variable name since we already have confirmed people. Otherwise we need to update the db.